src/core/stdc/stdio.d

```diff
@@ -963,42 +952,10 @@ else version( CRuntime_Microsoft )
     alias _vsnprintf vsnprintf;
 
     ///
-    uint _set_output_format(uint format);
-    ///
-    enum _TWO_DIGIT_EXPONENT = 1;
+    int _fputc_nolock(int c, FILE *fp);
```

Comment:
No, the implementation uses C++ and a lot of internals, so it doesn't seem easily translatable.